runtime-modules/project-token/src/lib.rs

```diff
@@ -803,6 +825,27 @@ decl_module! {
             Self::deposit_event(RawEvent::RevenueSplitLeft(token_id, member_id, staking_info.amount));
             Ok(())
         }
+
+        /// Allows to freeze or unfreeze this pallet. Requires root origin.
+        ///
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (1)`
+        /// - DB:
+        ///    - O(1) doesn't depend on the state or parameters
+        /// # </weight>
+        #[weight = WeightInfoToken::<T>::freeze_pallet()]
+        pub fn freeze_pallet(origin, freeze: bool) {
```

Comment:
```suggestion
        pub fn set_freezing_status(origin, freeze: bool) {
```
I would prefer this name (or similar ones possibly more expressive) , otherwise from the name it looks like you can set `frozen` state only

Reply:
@ignazio-bovo what you think about `set_frozen_status`?